R/vignette.R

```diff
@@ -39,6 +41,13 @@ use_vignette <- function(name, title = name) {
 use_article <- function(name, title = name) {
   check_is_package("use_article()")
 
+  deps <- desc::desc_get_deps(proj_get())
+  if (!"rmarkdown" %in% deps$package) {
+    ui_done("
+      Adding {ui_value('rmarkdown')} to {ui_field('Config/Needs/website')}")
+    use_description_list("Config/Needs/website", "rmarkdown")
```

Comment:
Since we're doing this here in `use_article()` specifically, we know enough to avoid adding markdown as an actual dependency in the case of an "articles only" package. I.e. then you only need it in `Config/Needs/website`.

Reply:
This seems worthy of a news bullet.